### PR TITLE
Fix import private key validation

### PR DESCRIPTION
**Pull request by lukachi**

### CODE REVIEW

File: packages/snap/src/index.ts [hunk 1]
```diff
@@ -220,7 +220,10 @@ export const onRpcRequest = async ({
 
       const params = request.params as CreateIdentityRequestParams;
 
-      if (params?.privateKeyHex && !utils.isHexString(params?.privateKeyHex)) {
+      if (
+        params?.privateKeyHex &&
+        !utils.isHexString(`0x${params?.privateKeyHex}`)
```
Comment (ihordiachenko) [inline packages/snap/src/index.ts:225]: Won't it pass when `privateKeyHex` is an empty string?

Comment (lukachi) [reply packages/snap/src/index.ts:225]: if there's no params?.privateKey, then it won't throw error. And then identity creation will use entropy

Comment (ihordiachenko) [reply packages/snap/src/index.ts:225]: But what if it's set to `''`? Could you check it manually, just to be sure?

Comment (lukachi) [reply packages/snap/src/index.ts:225]: but, '' is a false value

**Review by ihordiachenko — APPROVED** [review]

**Pull request merged**